sheets/utils.py

```diff
@@ -41,17 +41,25 @@ def generate_google_client_config():
 
 SpreadsheetSpec = namedtuple(
     "SpreadsheetSpec",
-    ["first_data_row", "last_data_column", "num_columns", "column_headers"],
+    [
+        "first_data_row",
+        "last_data_column",
+        "calculated_column_indices",
+        "num_columns",
+        "column_headers",
+    ],
 )
 coupon_request_sheet_spec = SpreadsheetSpec(
     first_data_row=GOOGLE_SHEET_FIRST_ROW + 1,
     last_data_column="H",
+    calculated_column_indices={7, 8},
```

Comment:
Where do these numbers come from? Is it possible to look up the indices from the column headers, or are there no column headers provided?

Reply:
I think it makes enough sense to move these indices to django settings. I'll make that change